src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -265,6 +265,10 @@ private static ListSelectionListener createDescriptionPanelUpdatingSelectionList
       final JList<String> gamesList, final List<DownloadFileDescription> maps, final MapAction action,
       final JLabel mapSizeLabelToUpdate) {
     return e -> {
+      if (e.getValueIsAdjusting()) {
+        return;
+      }
+
```

Comment:
I view this check as guarding the "normal path of execution" per [Replace Nested Conditional with Guard Clauses](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html), which I'm really OCD about (increased nesting and such). :smiley:

However, to be consistent, I could have done the same thing a few lines below when checking `somethingIsSelected`.  Since I didn't modify that line, I'll invert the condition on the line I added and nest it as you suggested.  But I bet that negative assertion is going to make it hard to fall asleep tonight. :smiley: